### ISSUE TITLE
inline const reference

### ISSUE THREAD

**Issue by ArabCoders** [phpDocumentor/phpDocumentor2#1718]
Hello,

Im trying to reference a constant inline, but for some reason it's not working as expected, it does create the link but it doesnt go there at all, for example:-

``` php
Interface tableXYZ
{
    const schema = [
        'id'            => 'int',
        'title'         => 'string',
        'text'          => 'string',
    ];

    /**
     * @param array $array
     *
     * @return self
     * @throws \Exception if array does not match {@see self::schema}
     */
    public function __construct( array $array );
}
```

i tried `{@see schema/self::schema/this:schema/$this:schema}` and none of them are working, it does kinda works if i added the fully Qualified Name eg `{@see \tables\xyz\tableXYZ::schema}`, i think it should works with self for referencing the class or interface it self..

**Comment by fi5er** [comment]
Possible duplicate of #1456

**Comment by jaapio** [comment]
Closed since #1456 is also resolved.
